src/controllers/UsersController.php

```diff
@@ -386,6 +386,38 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request Request
```

Comment:
Added https://joindin.jira.com/browse/JOINDIN-735 to be able to follow up on that